testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1674,3 +1693,49 @@
   node = get_target(target)
   node.run("iptables -F INPUT")
 end
+
+When(/^I generate the configuration "([^"]*)" of Containerized Proxy on the server$/) do |file_path|
+  # Doc: https://www.uyuni-project.org/uyuni-docs/en/uyuni/reference/spacecmd/proxy_container.html
+  command = "echo spacewalk > cert_pass && spacecmd -u admin -p admin proxy_container_config_generate_cert" \
+            " -- -o #{file_path} -p 8022 #{$proxy.full_hostname.sub('pxy', 'pod-pxy')} #{$server.full_hostname}" \
+            " 2048 galaxy-noise@suse.de --ca-pass cert_pass" \
+            " && rm cert_pass"
+  $server.run(command)
+end
+
+When(/^I set a new value in a configuration file$/) do |data_table|
```

Comment:
I don't like such generic steps, it looks like we are reinventing a programming language.

There must be a precise reason to change the configuration file. Why not express it in English ?

Reply:
I could not find occurrences of this new step in this PR. Is it a leftover ?

Reply:
Actually yes, it's a leftover 👍